Library/Formula/symlinks.rb

```diff
@@ -0,0 +1,23 @@
+class Symlinks < Formula
+  desc "scan/change symbolic links"
+  homepage "http://www.ibiblio.org/pub/Linux/utils/file/symlinks.lsm"
```

Comment:
Not that I've seen. But I could possibly link to Debian's package page,

- https://packages.debian.org/jessie/symlinks


CentOS has this, https://git.centos.org/blob/rpms!symlinks/4f7bcd8f06d12d8b704c57d7e24e3b53e0dcfab7/SPECS!symlinks.spec;jsessionid=h63096q1neq3w4hsnhg1p488 

and for Fedora I found this, http://rpm.pbone.net/index.php3/stat/4/idpl/31812403/dir/fedora_other/com/symlinks-1.4-12.fc23.x86_64.rpm.html

But I don't know if they have a page that actually has a description like Debian's.